### ISSUE TITLE
In paper, why loss term is added in the gradient part?

### ISSUE THREAD

**Issue by Rocky-CN** [tianpeiyang/PTF_code#6]
![image](https://user-images.githubusercontent.com/56460701/154925561-8bcb8bd5-5988-42d4-8378-50fee57b5f09.png)

Hi, I‘m a beginner of Reinforcement Learning. 

As the picture above illustrated ( Line 18 ), I wonder why  the cross-entropy loss term is added in  the process of calculating gradients.To my knowledge, the new auxiliary loss function is generally added to the existing loss function， instead of adding it when calculating the gradient.

Can you give a detailed explanation?

**Comment by tianpeiyang** [comment]
Hi, sorry to respond late.
Yes, you are correct that this should be $f(\beta_o,t)\nabla_{\theta'}L_H$ here. The code is correct. We will clarify this in the Arxiv version. Thank you.

**Comment by Rocky-CN** [comment]
Thank you very much, this answer is helpful !
